frontend/src/components/uam/helpers/get-rows-helper/get-rows.helper.ts

```diff
@@ -0,0 +1,19 @@
+import { UsersGetAllItemResponseDto } from 'common/types/types';
+
+import { UserTableAccessor } from '../../common/enums/enums';
+import { getTimeDifferenceWithCurrentTime } from '../helpers';
+
+const getRows = (
+  users: UsersGetAllItemResponseDto[],
+): UsersGetAllItemResponseDto[] => {
+  return users.map((user) => ({
+    [UserTableAccessor.ID]: user.id,
+    [UserTableAccessor.EMAIL]: user.email,
+    [UserTableAccessor.FULL_NAME]: user.fullName,
+    [UserTableAccessor.CREATED_AT]: getTimeDifferenceWithCurrentTime(
+      user.createdAt,
+    ),
```

Comment:
```suggestion
    [UserTableAccessor.CREATED_AT]: user.createdAt
```
you don't need to format the date for now